src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -108,14 +95,17 @@ public AndroidTargetConfiguration( ProcessPaths paths, InternalProjectConfigurat
 
         Path clangguess = Paths.get(this.ndk, "toolchains", "llvm", "prebuilt", hostPlatformFolder, "bin", "clang");
         this.clang = Files.exists(clangguess) ? clangguess : null;
+        if (projectConfiguration.getGraalVersion().compareTo(new Version("20.1.0")) > 0) {
```

Comment:
This fails for me, as I have installed `graalvm-ce-java11-20.2.0-dev`, and that produces ` GraalVM CE 20.2.0-dev`, that doesn't follow the current pattern (but this is a different issue though, I filed https://github.com/gluonhq/substrate/issues/617 for it).

Reply:
Can you check if it works for you now?